### ISSUE TITLE
Failed to remove user

### ISSUE THREAD

**Issue by atljoseph** [plandex-ai/plandex#124]
I’m running locally in docker compose. 

**_Finally_** figured out how to sign in locally but I messed up and want to start over. 

How does one do that? 

I get an error when trying to remove either user. 
- Cannot delete the only org owner <nil><nil>. 

Not super helpful. 
- Now I have 2 users and can’t delete either one. 

Can you ELI5 this for me?

**Comment by atljoseph** [comment]
I deleted the Postgres data dir and restarted the docker compose. It still lets me register a new user … even one with the same email as one already configured. Not sure why the users weren’t cleared out when Postgres was removed. But when I enter an org, which seems to be required (it complains), 500 error. 

Why is this so hard to get set up? 
Where is the secret button?

I’d like to be able to evaluate the product you guys put so much work into, and having trouble justifying spending more time on it.

**Comment by danenania** [comment]
Hey @atljoseph sorry to hear that it's not going smoothly for you.

One thing that could help for starting fresh is to delete the `.plandex` dir from the local directory (`rm -rf .plandex`), and then delete `.plandex-home` from your `$HOME` dir: `rm -rf ~/.plandex-home`

**Comment by danenania** [comment]
Actually if you're running in development mode, it would be `rm -rf .plandex-dev` and `rm -rf ~/.plandex-home-dev`

**Comment by atljoseph** [comment]
Thanks for the tip! can you pretty please put that in its own section and
script?

**Comment by atljoseph** [comment]
Start_local should maybe be accompanied by clean_local.

**Comment by danenania** [comment]
Yes, I'll add a note on that to the self-hosting guide and a `clean_local` script is a good idea as well. Did that resolve the issue for you?

**Comment by atljoseph** [comment]
Also if you would, the start local script always overwrites the env file. I
kept setting stuff in there and wondering why I was going crazy and it
disappeared. The bash should just only copy from _env if and only if it
does not already exist.

**Comment by danenania** [comment]
@atljoseph Ok, I updated the start_local.sh script to not overwrite `.env` if it already exists.

**Comment by atljoseph** [comment]
Thanks. Another going crazy moment. Why is pdxd STILL persisting even after I update the env var? Seems like it defeats the purpose of there being env vars!?

By the way, landed is way easier to type than pdxd. I’m trying to use plx. 

![image](https://github.com/plandex-ai/plandex/assets/25017621/b0c8eb23-d70e-41ae-a052-599c362c3c37)

**Comment by atljoseph** [comment]
Ok I got it. That dev.sh script is just over complicated. Go install is so elegant.
